labcodes/labstorybook
# Playgrounds improvements

**Link to task: https://labcodes.atlassian.net/browse/CSYS-191**

**Staging app:** https://playgrounds-improvements--labstorybook-master.netlify.app/

**How to test:**
- Check if all Toggles in components playgrounds are wrapped with `<fieldset>` using `<legend>`
- Check if there are playgrounds still using `<br />` (they shouldn't)
- Check if controls perform as expected in all playgrounds
- Check if the layout crashes in any breakpoint for all playgrounds

**Description of your solution:**
- Added styles for `<h4>`, `<legend>`, `<label>`, and `<select>`
- Added a class to display controls inline 
- Organized code in stylesheet
- Updated `<fieldset>` use to alway have a `<legend>` assigned
- Wrapped all toggle controls with a `<fieldset>` and `<legend>`
- Updated some control types (input to select) and styles
